IFC4x3/Properties/g/GasType_3xONOgIiDFaPnwg2ne4hqt/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GasType_3xONOgIiDFaPnwg2ne4hqt" Name="GasType" UniqueId="fb61762a-4ac3-4f91-9c7a-a82c6812bd37" PrimaryDataType="IfcText" />
```

Comment:
`PrimaryDataType` should probably be changed to `IfcLabel` or `IfcIdentifier`.

Reply:
label is good